rewrite-maven/src/test/java/org/openrewrite/maven/UpgradeDependencyVersionTest.java

```diff
@@ -2008,4 +2009,45 @@ void exactVersionMissingInMavenMetadata() {
           )
         );
     }
+
+    @Test
+    void upgradeVersionWithGroupIdAndArtifactIdDefinedAsMavenConfigProperty() {
+        rewriteRun(
```

Comment:
This test fails at the moment, because the properties in the text LST are not recognised:

![image](https://github.com/user-attachments/assets/b3b5a51c-186d-405b-85ab-f1afed29b710)